docs/src/examples/components/RadioGroup/Types/RadioGroupExample.shorthand.tsx

```diff
@@ -29,11 +12,43 @@ class RadioGroupExample extends React.Component {
         <Divider />
         <RadioGroup
           defaultCheckedValue="capricciosa"
-          items={items}
+          items={this.getItems()}
           checkedValueChanged={this.handleChange}
         />
       </div>
     )
   }
+
+  getItems() {
+    return [
+      { name: 'pizza', key: 'Capricciosa', label: 'Capricciosa', value: 'capricciosa' },
+      {
+        name: 'pizza',
+        key: 'Prosciutto',
+        label: 'Prosciutto',
+        value: 'prosciutto',
+        disabled: true,
+      },
+      {
+        name: 'pizza',
+        key: 'Custom',
+        label: (
+          <Text>
+            Choose your own{' '}
+            <Input input={{ tabIndex: this.state.inputTabIndex }} inline placeholder="flavour" />
```

Comment:
I am bit confuse here... Can we still use the input shorthand, now that we don't have it in the Input component?

Reply:
Watch out, this definition that you have is different then having 
```
label: {
  content: (
    <Text>
      Choose your own{' '}
      <Input input={{ tabIndex: this.state.inputTabIndex }} inline placeholder="flavour" />
     </Text>
  )
}
``` 
With your definition you are just having this component rendered, so any default or overridden props that you have for the label component will not be applied.

Reply:
> I am bit confuse here... Can we still use the input shorthand, now that we don't have it in the Input component?
no, this will work correctly once we fix the input shorthand, but it does no harm now, just key nav is not working correctly for the case when we tab from an item to the next component

> With your definition you are just having this component rendered, so any default or overridden props that you have for the label component will not be applied.

I don't get it, that's what I want.. the label slot is already inside `Label` component

Reply:
> I don't get it, that's what I want.. the label slot is already inside `Label` component

Sorry, my mistake, I thought this contain `Label.create(label ...` It is defined as it should.

Reply:
Let's get rid of the input shorthand, and we will add it in the Input's PR